Formula/libadwaita-1.rb

```diff
@@ -0,0 +1,36 @@
+class Libadwaita1 < Formula
```

Comment:
1 is the version right? Then it should not be in the name.

Reply:
The newest version is always unversioned by when a 2.0 exists we can rename this formula@1.

GTK is a different situation since it's essentially different software once they bump the version (and legacy reasons)

Reply:
Ok then, I dropped the explicit version from the formula name. Thanks for your explanation.